myhdl/_Cosimulation.py

```diff
@@ -137,7 +137,10 @@ def _get(self):
                     if next >= (1 << (s._nrbits-1)):
                         next |= (-1 << s._nrbits)
             except ValueError:
-                next = intbv(0)
+                if v in 'xX':
+                    next = s._init
```

Comment:
In Co-simulation, icarus may return 'x', corresponding to None in MyHDL. Without the change, 'x' will be translated to intbv(0). Tristate bus relies on the rule that all but one driver's value are None, so assigning zero values to it does not work. My first attempt was:

```python
if v in 'xX':
  next = None
else:
  next = intbv(0)    # probably deadcode
```

However, the logical and arithmetic operators defined for `_Signal` do not support None value. The above code causes many test cases to fail. Not wanting to code "None support" for all operators, I'm using the fact that `_TristateSignal._init == None` while other types of signals usually have a defined initial value. This way, all previous working code hopefully will continue to work, and tristate bus correctly receives None values as it should.

This is clearly not perfect solution. I think `_Signal` operators should support None values, but being new to MyHDL, I'm not sure if this is the way to go.